src/CostManagement/docs/Get-AzCostManagementExport.md

```diff
@@ -0,0 +1,175 @@
+---
+external help file:
+Module Name: Az.Cost
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cost/get-azcostmanagementexport
+schema: 2.0.0
+---
+
+# Get-AzCostManagementExport
+
+## SYNOPSIS
+The operation to get the export for the defined scope by export name.
+
+## SYNTAX
+
+### List (Default)
+```
+Get-AzCostManagementExport -Scope <String> [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+### Get
+```
+Get-AzCostManagementExport -Name <String> -Scope <String> [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+### GetViaIdentity
+```
+Get-AzCostManagementExport -InputObject <ICostIdentity> [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The operation to get the export for the defined scope by export name.
+
+## EXAMPLES
+
+### Example 1: Get all cost management exports under a subscription
+```powershell
+PS C:\> Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f"
+
+Name        Type
+----        ----
+TestExport  Microsoft.CostManagement/exports
+TestExport1 Microsoft.CostManagement/exports
+```
+
+this command gets all cost management exports under a subscription.
+
+### Example 2: Get a cost management export by name
+```powershell
+PS C:\> Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -Name 'TestExport'
+
+Name       Type
+----       ----
+TestExport Microsoft.CostManagement/exports
+```
+
+This command gets a cost management export by name.
+
+### Example 3: Get a cost management export by object
+```powershell
+PS C:\> $export = Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -Name 'TestExport'
+PS C:\> Get-AzCostManagementExport -InputObject $export
+
+Name       Type
+----       ----
+TestExport Microsoft.CostManagement/exports
+```
+
+This command gets a cost management export by object.
+
+### Example 4: Get a cost management export by pipeline
+```powershell
+PS C:\> @{Id = 'subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f/providers/Microsoft.CostManagement/exports/TestExport' } | Get-AzCostManagementExport
+
+Name       Type
+----       ----
+TestExport Microsoft.CostManagement/exports
+```
+
+This command gets a cost management export by pipeline.
```

Comment:
Sorry, I think this example is not necessary for this cmdlet.

Reply:
Done